class-03-jquery-and-events/pair-assignment/starter-code/scripts/articleView.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+// IN-CLASS TODID: Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
+var articleView = {
+};
+
+articleView.populateFilters = function() {
+  console.log(1);
+  $('article').not('.template').each(function() {
```

Comment:
nicely done! you could also get these values from the article objects in the articles array if you wanted.